src/main/java/org/fundacion/jala/converter/models/Project.java

```diff
@@ -7,13 +7,15 @@
  *
  * @author Juan Pablo Gonzales
```

Comment:
Add last name of mother to Juan Pablo Gonzalez Alvarado

Reply:
ok thanks Gus